PostHog/posthog
# feat: display entire element chain

## Problem

Continuing work from #16126 which aims to resolve #14347 

I'll still need to fix the tests but I've tried to replicate the desired results requested for #16126 

## Changes
_I apologize for the slight stutter in the video, my computer is struggling_

https://github.com/PostHog/posthog/assets/107593248/85d3e42d-5434-4c7f-95e7-5ddfa41a71d5

https://github.com/PostHog/posthog/assets/107593248/5191f434-1538-4f86-9f83-70402e0d6931

## How did you test this code?

Manual Testing:

- Expanded multiple events to test various lengths of event captures.
- Tested Toolbar view when editing an action
- Clicked the new "Show x parent elements (x elements hidden)" row to verify the element text was captured correctly.
- Verified functionality between every commit.
- Double checked all of the pain points mentioned in #16126 to ensure those bugs were addressed.
